plugins/CoreHome/Columns/IdSite.php

```diff
@@ -23,11 +27,13 @@ class IdSite extends VisitDimension
     // INDEX(idsite, config_id, visit_last_action_time) and we maybe not be sure whether config_id already exists at
     // installing point (we do not know whether visit_last_action_time or idsite column would be added first).
 
+    protected $nameSingular = 'General_Measurable';
+    protected $namePlural = 'General_Measurables';
     protected $type = self::TYPE_TEXT;
 
     public function getDbColumnJoin()
     {
-        return new SiteNameJoin();
+        return new SiteIdJoin();
```

Comment:
Considering it is a idsite already does it need to be joined? or is it there to ignore removed sites?

Reply:
I assumed it was necessary (I'm not familiar with this code). SiteNameJoin joins on idsite, but selects the name, figured I'd need another join, but I guess this join is just for extra data during aggregation?

I guess we can select for sites that aren't deleted, I'm not sure if it makes sense. If a rollup has a deleted site, then it should be fixed in the rollup, not ignored, right?

Reply:
True, when a site is deleted, it should be also removed from the roll-up. And when using this dimension for an individual site (not for a roll-up), and this site is deleted, this report would not be archived anymore. It would be only a problem if for some reason the site was not removed from the roll-up automatically.

The join is not needed but can be used when resolving extra data like `idaction_url` wants to be resolved to `log_action.name`. 

This method to get a join `getDbColumnJoin()` is also used by GDPR features (in a PR) to fetch all data stored for a visitor / data subject. 

This join is ignored though so we can probably just remove the join. It would be only needed if we directly wanted to get the name of the site during archiving.

Reply:
Gotcha.